app/assets/v2/js/pages/dashboard.js

```diff
@@ -86,6 +86,7 @@ var buildURI = function() {
  */
 var save_sidebar_latest = function() {
   localStorage['order_by'] = $('#sort_option').val();
+  localStorage['order_by_date'] = '-expires_date';
```

Comment:
Doesn't that always ensure we query with `-expires_date` ?

Reply:
@pinkiebell yes it does, is that an issue?

Reply:
Not if that is intended 😄 cc @mbeacom ?